packages/frontend/src/Components/Search.tsx

```diff
@@ -15,9 +15,12 @@ const Search = () => {
         // TODO: Phase 2: Make the Search API call to our node backend
         // NOT the actual spotify API
         // the node backend will call the spotify API
+        axios.get(`/myApi/search?q=${searchText}`).then((res) => {
+          setSearchResults(res.data);
+          setLoading(false);
+        });
```

Comment:
Add .catch((error) => console.log(error)); to catch any errors from he axios call